docs/RELEASE_NOTES.md

```diff
@@ -1,4 +1,22 @@
 # Release Notes
+## v5.4
+_April 2020:_ Version 5.4
+This release includes the following improvements:
+
+- A new contact loader for loading contacts in straight from NGP VAN
```

Comment:
do we want to label this as experimental until we've seen a deploy (and we have the action handlers pieces).  Otherwise, it would be good to link to documentation

Reply:
Yeah that's a great idea. Do we have documentation on this specific contact loader yet?